src/lib/OpenEXRUtil/ImfFlatImageChannel.h

```diff
@@ -20,8 +20,8 @@
 #include "ImfUtilExport.h"
 #include "ImfImageLevel.h"
 
-#include <ImfPixelType.h>
-#include <ImfFrameBuffer.h>
+#include "ImfPixelType.h"
+#include "ImfFrameBuffer.h"
 #include <ImathBox.h>
 #include <half.h>
```

Comment:
shouldn't these be

    <Imath/ImathBox.h>

ec?

Reply:
That would seem reasonable, but our CMake files must not be set up that way.  The compile flags have -I_build/_deps/imath-src/src/Imath. The Imath headers only end up in the Imath subfolder after the install step. During the build, they're referenced out of the cloned source directory.

If you configure cmake with -DCMAKE_INSTALL_PATH=<imath install>, then it works for the OpenEXR headers to include <Imath/ImathBox.h>, because the cflags have -I<imath install>/include.